src/bot.jl

```diff
@@ -32,11 +32,13 @@ include("bot/snoopi_bench.jl")
 
 # deprecation and backward compatiblity
 macro snoopiBot(args...)
-    warning("@snoopiBot is deprecated. Use @snoopi_bot instead")
-    return esc(:(@snoopi_bot($(args...))))
+     f, l = __source__.file, __source__.line
+     depwarn("`@snoopiBot` at $f:$l is deprecated, rename the macro to `@snoopi_bot`.", Symbol("@snoopiBot"))
```

Comment:
Have you tested this? `depwarn` isn't exported from Base, so I expect this will throw an error.

The `@eval @deprecate` probably doesn't do anything if you are also defining the macro directly.

Reply:
Thank you. Now it works:
![image](https://user-images.githubusercontent.com/16418197/73923272-3894fc80-4890-11ea-8a70-62093735b66b.png)